packages/headless/src/controllers/facets/range-facet/date-facet/date-range.ts

```diff
@@ -2,29 +2,59 @@ import dayjs from 'dayjs';
 import utc from 'dayjs/plugin/utc';
 import customParseFormat from 'dayjs/plugin/customParseFormat';
 import {DateRangeRequest} from '../../../../features/facets/range-facets/date-facet-set/interfaces/request';
+import {FacetValueState} from '../../../../features/facets/facet-api/value';
 
 dayjs.extend(utc);
 dayjs.extend(customParseFormat);
 
-type DateType = string | number | Date;
+export type DateValueType = string | number | Date;
```

Comment:
Any suggestions around the name of this type alias? I felt `DateType` was too general, but `DateRangeValueType` was too confusing.

Reply:
I'm not particularly fond on the word `Type` in something that is... a type.

Suggestions: `DateInput`, `DateConfig`.

As in, you use this to "input" or "configure" a date.